build/assets/scripts/high-performance-hooks.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+pid=$(jq '.pid' /dev/stdin)
+mask=$1
+ns=$(ip netns identify "${pid}")
+
+[[ -n "${pid}" ]] && [[ -n "${mask}" ]] && [[ -n "${ns}" ]] || exit 0
+
+ip netns exec "${ns}" mount -o remount,rw /sys
```

Comment:
If this fails there is no point in continuing. OTOH if we do we should "only" get a bunch of errors somewhere

Reply:
yep, I better bail out [no yet though about tightening things up]

Reply:
I will add a "set -e" to the script to bail out if anything goes wrong, would be good enough?

Reply:
using `set -e` is a good idea in general, but it seems our case it's a bit more complex.
let's consider the last three commands, which is the real deal here.
1. if remounting sysfs rw fails, we do want to bail out (but keep doing will do no harm besides wasted work)
2. if find fails, we still want to remount sysfs ro!
3. if remounting sysfs ro fails, we have no way to recover anyway so nothing to do here

Reply:
right, I'll be more precise